dhall-to-cabal.cabal

```diff
@@ -13,6 +13,7 @@ executable dhall-to-cabal
         Cabal -any,
         dhall -any,
         text -any,
-        containers -any
+        containers -any,
+        optparse-applicative -any
```

Comment:
*gasp* Did you directly update the cabal file!? :wink: Need this in `dhall-to-cabal.dhall`. I'll add that now.

Reply:
haha oh dear - sorry that was silly ;-)

Reply:
Not to worry - thanks for your contribution!